packages/@react-aria/searchfield/src/useSearchField.ts

```diff
@@ -102,6 +102,8 @@ export function useSearchField(
     clearButtonProps: {
       'aria-label': formatMessage('Clear search'),
       excludeFromTabOrder: true,
+      // @ts-ignore
```

Comment:
because this is only used internally?

Reply:
yeah I guess we didn't include it in useButton because we didn't want people to use it in general 😂